cmd/generate-bindings/bindings/bindgen.go

```diff
@@ -81,15 +84,28 @@ func GenerateBindings(
 		return errors.New("must provide either combinedJSONPath or abiPath")
 	}
 
-	// Generate w/ forked abigen
+	// Generate regular bindings w/ forked abigen
 	outSrc, err := abigen.BindV2(types, abis, bins, pkgName, libs, aliases, tpl)
 	if err != nil {
 		return fmt.Errorf("BindV2: %w", err)
 	}
 
-	// Write file
+	// Write regular bindings file
 	if err := os.WriteFile(outPath, []byte(outSrc), 0o600); err != nil {
 		return fmt.Errorf("write %q: %w", outPath, err)
 	}
+
+	// Generate mock bindings
+	mockSrc, err := abigen.BindV2(types, abis, bins, pkgName, libs, aliases, mockTpl)
+	if err != nil {
+		return fmt.Errorf("BindV2 mock: %w", err)
+	}
+
+	// Write mock file with "_mock.go" suffix
+	mockPath := strings.TrimSuffix(outPath, ".go") + "_mock.go"
```

Comment:
Can we put this in a mock subfolder? I don't know how much of it will be able to be isolated from the build otherwise.